lib/owl_plot.ml

```diff
@@ -1170,6 +1170,62 @@ let pie ?(h=_default_handle) ?(spec=[]) x =
   p.plots <- Array.append p.plots [|f|];
   if not h.holdon then output h
 
+let _range_ i j =
```

Comment:
Thanks for pointing it out. It turns out I already used the `Owl_dense_matrix.D.linspace` and forgot to delete the initial implementation of `range` :)